var/spack/repos/builtin/packages/py-wandb/package.py

```diff
@@ -10,23 +10,49 @@ class PyWandb(PythonPackage):
     """A tool for visualizing and tracking your machine
     learning experiments."""
 
-    homepage = "https://github.com/wandb/"
-    url = "https://github.com/wandb/client/archive/v0.10.1.tar.gz"
+    homepage = "https://github.com/wandb/wandb"
+    pypi = "wandb/wandb-0.13.9.tar.gz"
 
-    version("0.10.1", sha256="abd334cd1460ac1f6e5aa959d3e04c46cd246f96cfc3323fc0572916760d32ab")
+    maintainers = ["thomas-bouvier"]
 
+    version("0.13.9", sha256="0a17365ce1f18306ce7a7f16b943094fac7284bb85f4e52c0685705602f9e307")
+    version("0.10.1", sha256="d02427cda58a6618ba10a027a76d9e3f68ad923d35964b1b68785c49e5160009", deprecated=True)
+
+    depends_on("python@3.6:", type=("build", "link", "run"), when="@0.13.0:")
     depends_on("py-setuptools", type="build")
+
+    with when("@0.13.0:"):
+        depends_on("py-grpcio@1.46.3:", type="build")
+        depends_on("py-grpcio-tools@1.46.3:", type="build")
+        depends_on("py-twine", type="build")
+        depends_on("py-pathtools", type=("build", "run"))
+        depends_on("py-setproctitle", type=("build", "run"))
+        depends_on("py-appdirs@1.4.3:", type=("build", "run"))
+        depends_on("py-protobuf@3.12.0:4", type=("build", "run"), when="^python@:3.8 platform=linux")
+        depends_on("py-protobuf@3.15.0:4", type=("build", "run"), when="^python@3.9 platform=linux")
+        depends_on("py-protobuf@3.19.0:4", type=("build", "run"), when="^python@3.10: platform=linux")
```

Comment:
You can add:
```python
conflicts("^py-protobuf@4.21.0")
```
to handle the `!=`, or add it to the version range in `depends_on`: `3.12:4.20,4.21.1:4`